### PR TITLE
Stage 1

### PR DESCRIPTION
**Pull request by Moniarchy**

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+div {
+  border: grey;
```
Comment (shakalee14) [inline styles/style.css:6]: does border work with 1 argument?

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+}
+
+.calculator-answer-screen {
+  align-items: flex-end;
+  background-color: #a0a0a0;
+  border-top-left-radius: 7px;
+  border-top-right-radius: 7px;
+  color: white;
+  display: flex;
+  font-style: thin;
```
Comment (shakalee14) [inline styles/style.css:54]: i think you can do font-weight too as lighter.

**Review by shakalee14 — COMMENTED** [review]
LGTM!

File: styles/style.css (hunk 2 above)
Comment (Moniarchy) [reply styles/style.css:54]: I tried to and it didn't work for me. :(

File: styles/style.css (hunk 1 above)
Comment (Moniarchy) [reply styles/style.css:6]: Yes! Does it look weird?

**Pull request merged**